spec/models/ability_spec.rb

```diff
@@ -145,7 +145,7 @@
   end
 
   it 'should allow users to crud events they created' do
-    event = Event.new(owner: @user)
+    event = FactoryBot.create(:event, :has_dates, owner_id: @user.id)
```

Comment:
This is awkward because it's not really obvious why the even needs to have dates, I'd rather see this in a sperate ability test only for Tournaments and leagues, because rankinglists don't have any dates. Also adding another test with the other case (when you cannot delete the event) would be a good idea imo.

Reply:
Users can only delete events now under certain conditions so this test has to be changed. But I agree that the event object having a date here is weird. I will change this test and create a separate one for the changed delete ability.